### PR TITLE
Adds cloud advisory on all install pages that have proxy/auth install

### PR DESCRIPTION
**Pull request by stevenGravy**
This advisory was on the linux server getting started.  Applicable to digital ocean, docker-compose and enterprise getting started.

image

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/includes/cloud/cloudmanagedadvisory.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<Details scopeOnly={true} scope={["cloud"]} title="Teleport Cloud">
+This guide deploys the Teleport Auth Service and Proxy Service, which are fully managed in Teleport Cloud. If you are a Teleport Cloud customer, we recommend following our guides for deploying specific services on your Teleport Nodes. Here are some examples:
```
Comment (ptgott) [inline docs/pages/includes/cloud/cloudmanagedadvisory.mdx:2]:
```suggestion
This guide deploys the Teleport Auth Service and Proxy Service, which are fully managed in Teleport Cloud. If you are a Teleport Cloud customer, we recommend following our guides for accessing specific resources within your infrastructure. Here are some examples:
```

I wrote this when I mistakenly conflated "Nodes" with "agents," so I've added a suggestion.

File: docs/pages/includes/cloud/cloudmanagedadvisory.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- [Application Access](../application-access/getting-started.mdx)
+- [Database Access](../database-access/getting-started.mdx)
+- [Server Access](../server-access/getting-started.mdx)
```
Comment (ptgott) [inline docs/pages/includes/cloud/cloudmanagedadvisory.mdx:6]: Should we go ahead and list Kubernetes Access and Desktop Access here? Then we can remove "Here are some examples" above and replace the final "." in the preceding paragraph with a ":".

**Review by r0mant — APPROVED** [review]
lgtm when Paul's comment is addressed

File: docs/pages/includes/cloud/cloudmanagedadvisory.mdx (hunk 2 above)
Comment (stevenGravy) [reply docs/pages/includes/cloud/cloudmanagedadvisory.mdx:4]: done

**Pull request merged**